src/node/rpc/test/frontend_test.cpp

```diff
@@ -506,16 +473,22 @@ TEST_CASE("No certs table")
   CHECK(response[jsonrpc::RESULT] == true);
 }
 
-// We need an explicit main to initialize kremlib and EverCrypt
-int main(int argc, char** argv)
+TEST_CASE("Signed read requests can be executed on backup")
 {
-  doctest::Context context;
-  context.applyCommandLine(argc, argv);
-  ::EverCrypt_AutoConfig2_init();
-  int res = context.run();
-  if (context.shouldExit())
-    return res;
-  return res;
+  prepare_callers();
```

Comment:
Do we want to check here that the signature has not been stored in the kv?

Reply:
Using the `BackupStubConsensus`, it is guaranteed that the transaction will not commit as `replicate()` will fail. If it did, we would get a `TX_FAILED_TO_REPLICATE` in the response.